arch/xtensa/src/esp32/hardware/esp32_soc.h

```diff
@@ -178,6 +178,7 @@
 
 /* Other interrupt numbers should be managed by the user */
 
+#define DR_REG_SHA_BASE             0x3ff03000
```

Comment:
You can remove it. This macro is defined at `esp-hal-3rdparty/components/soc/esp32/include/soc/soc.h`. Just include `#include "soc/soc.h"`

```suggestion
```